README.md

```diff
@@ -173,6 +172,12 @@ We stopped doing Swift-2.3-specific branches (too much maintenance work), so if
  * or fork the repo just to change the `SWIFT_VERSION` build setting to `2.3`
  * or ask Carthage maintainers to [fix this issue](https://github.com/Carthage/Carthage/issues/1445) once and for all.
 
+We stopped doing Swift-3.x-specific branches (too much maintenance work), so if you want Carthage to build the framework with Swift 3.x you can:
+
+ * either use an older Xcode version
+ * or use the previous version of `OHHTTPStubs` (6.2.0) — whose `master` branch uses `3.0`
+ * or fork the repo just to change the `SWIFT_VERSION` build setting to `3.0`
+
```

Comment:
Might also add well remove the paragraph about 2.3 too here and make that paragraph about 3.x just replace it (instead of being in addition of it).

Also I'm not sure we did Swift-3.x branches back in the day, branches was just used during the 2 to 3 transition. So might just as well start that paragraph directly with "If you want…"

Reply:
Updated in c9b28c9.